frontend/src/lib/components/PropertyKeyInfo.tsx

```diff
@@ -376,15 +380,31 @@ export const keyMapping: KeyMappingInterface = {
             examples: ['Australia/Sydney', 'Asia/Kolkata'],
         },
         $geoip_subdivision_1_name: {
-            label: 'Subdivision Name',
+            label: 'Subdivision 1 Name',
             description: `Name of the subdivision matched to this event's IP address.`,
             examples: ['New South Wales', 'Tamil Nadu'],
         },
         $geoip_subdivision_1_code: {
-            label: 'Subdivision Code',
+            label: 'Subdivision 1 Code',
             description: `Code of the subdivision matched to this event's IP address.`,
             examples: ['NSW', 'TN'],
```

Comment:
For examples I would add a US example, maybe I'm biased but I think there's a good grasp of states/cities using the US as a reference.

Reply:
Added an example obtained from a NY IP address